### ISSUE TITLE
Segmentation fault when calling scene.addObject from a component

### ISSUE THREAD

**Issue by pqftgs** [UPBGE/blender#86]
Tested with build 7ba5e25
Example blend http://www.pasteall.org/blend/41479

This Python component will crash the engine
```
import bge


class Test(bge.types.KX_PythonComponent):
    def start(self, args):
        object = self.object
        scene = object.scene
        scene.addObject('MyObject', object)
```

**Comment by youle31** [comment]
Hi! I think it has to be done in update() function or another function in the component. start() is just here to initialize class attributes. But yeah, we should fix segfault and display an error message when we try to to inappropriate stuff in start() method. However, as I didn't work on pycomponents, I leave this task to panzergame or TwisterGE.

**Comment by pqftgs** [comment]
This happens in update too but used start in my example to simplify the code. (That said, it would be nice if this sort of thing was also supported in start(), but I can work around that if absolutely necessary)

**Comment by youle31** [comment]
Ah ok, so I let the experts deal with this bug :)

**Comment by panzergame** [comment]
Fixed in the last merge the BFBGE commit : 2050ecc307de9159b50c7f9798e47a366f505f2e
